app/controllers/airlines_controller.rb

```diff
@@ -2,7 +2,8 @@ class AirlinesController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
 
   def index
-    @airlines = Airline.page(params[:page]).per(24)
+    airlines = Airline.all.sort { |x, y| y.average_rating <=> x.average_rating }
```

Comment:
Could this be extracted into at least a class method? This is a lot of method chaining for the controller and its not super clear/easy to understand what the code is trying to do.